modules/nodes-nodes-working-deleting-bare-metal.adoc

```diff
@@ -31,14 +31,18 @@ $ oc adm cordon <node_name>
 $ oc adm drain <node_name> --force=true
 ----
 
+. If the node is unresponsive, power down the physical hardware to avoid data corruption.
```

Comment:
```suggestion
. If the node is unresponsive it may still be running a workload that writes to shared storage. Power down the physical hardware to avoid data corruption.
```

I find out useful to explain why data corruption is possible unless we power off

Reply:
Thanks, Dan. Updated per your suggestion.